playbooks/container-runtime/private/setup_storage.yml

```diff
@@ -19,3 +19,7 @@
       when:
         - container_runtime_docker_storage_type|default('') == "overlay2"
         - openshift_docker_is_node_or_master | bool
+    - include_role:
```

Comment:
Remove the with_items from here and use import_role.  Move the with_items to each task in the next file.  include_role is not performant.